tests/unit/core/test_utils.js

```diff
@@ -540,6 +542,17 @@ describe('isAllowedOrigin', () => {
   });
 });
 
+describe('getIconUrl', () => {
+  const addon = { ...fakeAddon };
+
+  it('return icon url as in fake addon', () => {
+    expect(getIconUrl(addon)).toEqual('https://addons.cdn.mozilla.net/webdev-64.png');
```

Comment:
For a test like this I think it helps to input a specific value that you're depending on even though the default value is sufficient. This makes the test very explicit and easier to understand. Example:

```js
const allowedIcon = 'https://addons.cdn.mozilla.net/any-icon.png';
expect(getIconUrl({ ...fakeAddon, icon_url: allowedIcon })).toEqual(allowedIcon);
```